runtime/bindings/python/src/compatibility/ngraph/utils/node_factory.py

```diff
@@ -39,9 +41,24 @@ def create(
 
         @return   Node object representing requested operator with attributes set.
         """
+        if arguments is None and attributes is None:
+            node = self.factory.create(op_type_name)
+            node._attr_cache = {}
+            node._attr_cache_valid = False
+            return node
+
+        if arguments is None and attributes is not None:
+            raise UserInputError(
+                'Error: cannot create "{}" op without arguments.'.format(
+                    op_type_name
+                )
```

Comment:
Can you put a test to check if this error is raised correctly?

Reply:
New test added `test_node_factory_validate_missing_arguments()`